criteria/document-objectives.md

```diff
@@ -6,14 +6,16 @@ order: 7
 
 ## Requirements
 
-* The codebase MUST contain documentation on its objectives – like a mission and goal statement – that is understandable for designers and developers so that they can use or contribute to the codebase
+* The codebase MUST contain documentation of its objectives – like a mission and goal statement – that is understandable by designers and developers so that they can use or contribute to the codebase
 * The codebase SHOULD contain documentation on its objectives understandable by policy makers and management
-* The codebase MAY contain documentation on its objectives for the general public
+* The codebase MAY contain documentation of its objectives for the general public
 
 ## Why this is important
 
-* Provide an easy way for people to decide whether this codebase is interesting for them now or in the future
-* Help scope your own development as well as clearly communicate to other stakeholders and contributors what the codebase is for
+Documenting your objectives:
```

Comment:
This is good, however can be replaced by adding 'it' to the bullets as well (if preferred)